lightningd/invoice.c

```diff
@@ -173,6 +174,10 @@ invoice_payment_hook_cb(struct invoice_payment_hook_payload *payload,
 	struct invoice invoice;
 	enum onion_type failcode;
 
+	/* We notify here to benefit from the payload and because the hook callback is
+	 * called even if the hook is not register. */
```

Comment:
Grammar nazi nit: even if the hook is not register**ed**.

Reply:
Haha thanks